src/loose/oose/fis/lab/student/manager/controllers/LoginController.java

```diff
@@ -0,0 +1,39 @@
+package loose.oose.fis.lab.student.manager.controllers;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.PasswordField;
+import javafx.scene.control.TextField;
+import javafx.scene.text.Text;
+
+public class LoginController {
+    @FXML
+    public Text loginMessage;
+    @FXML
+    public PasswordField passwordField;
+    @FXML
+    public TextField usernameField;
+
+    @FXML
+    public void handleLoginButtonAction(){
+        String username = usernameField.getText();
+        String password = passwordField.getText();
+
+        if( username == null || username.isEmpty()){
+            loginMessage.setText("Please type in a username!");
+            return;
+        }
+        if( password == null || password.isEmpty()){
+            loginMessage.setText("Password cannot be empty!");
+            return;
+        }
+        if(username.equals("student") && password.equals("student")){
+            loginMessage.setText("Logged in as a student!");
+            return;
+        }
+        if(username.equals("teacher") && password.equals("teacher")){
+            loginMessage.setText("Logged in as a teacher!");
+            return;
+        }
+        loginMessage.setText("Incorrect login!");
+    }
```

Comment:
If login as a teacher succeeds, the user should be taken to the view-students scene